.github/workflows/lock.yml

```diff
@@ -11,23 +11,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Lock thread
-        uses: dessant/lock-threads@v2
+        uses: dessant/lock-threads@v3
         with:
           # Issue specific options
-          issue-lock-inactive-days: 30 # 30 days before locking
-          issue-exclude-labels: "Do not lock this"
-          issue-lock-comment: >
+          issue-inactive-days: 30 # 30 days before locking
+          exclude-any-issue-labels: "Do not lock this"
+          issue-comment: >
             This issue has been automatically locked since there
             has not been any recent activity after it was closed.
             Please open a new issue for related bugs
-          issue-lock-labels: "Outdated and locked"
+          add-issue-labels: "Outdated and locked"
           # PR specific options
-          pr-lock-inactive-days: 60 # wait 2 months aprox.
-          pr-exclude-labels: "Do not lock this"
-          pr-lock-comment: >
+          pr-inactive-days: 60 # wait 2 months aprox.
+          exclude-any-pr-labels: "Do not lock this"
+          pr-comment: >
             This pull request has been automatically locked since there
             has not been any recent activity after it was closed.
             Please open a new issue for related bugs.
-          pr-lock-labels: "Outdated and locked"
+          add-pr-labels: "Outdated and locked"
```

Comment:
I had to modify these values, to avoid warnings from the action and a degraded behavior.